sql_metadb/derived_tables/holdings_administrative_notes.sql

```diff
@@ -0,0 +1,26 @@
+-- This derived table extracts administrative notes from the holdings record.
+DROP TABLE IF EXISTS holdings_administrative_notes;
+
+CREATE TABLE holdings_administrative_notes AS 
+SELECT 
+    hd.instance_id,
+    hd.id AS holdings_id,
+    hd.hrid AS holdings_hrid,
+    adminNotes.jsonb #>> '{}' AS administrative_notes,
```

Comment:
Advised to use singular column name and all-lowercase table names
```suggestion
    admin_notes.jsonb #>> '{}' AS administrative_note,
```